app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,12 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
why is it twice?

Reply:
I had troubles implementing bootstrap correctly and forgot to delete this line from one of the things I tried